### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by appiekap653** [Cleanarr/Cleanarr#3] [Ideas]
Since the project is in early stage, I welcome all kinds of feature requests.

The Readme shows a basic description of the features being implemented but if someone wants something different or new in the software, we can easily implement them in this early stage of development.

So if you have ideas, requests or questions about features please feel welcome to post them here.

With regards,

Arjen Hissink

**Comment by skinnyzaz** [comment]
The ability to prefer hevc over h264 or the other way around.

A lot of my duplicates have a hevc version and I would like to keep the hevc and delete the h264. Currently it seems to always pick the hevc file to delete by default.

Thanks for the script, loving it so far.

**Reply by appiekap653** [reply]
I don't think this program is the right place for this feature. The main purpose of this program is to remove movies and series that have been watched. Not to delete them if there is an upgrade available. If you use sonarr and radarr you can set it up to just exactly do what you need. It can delete files when it picks an upgrade or proper version. Since those programs are intended to add movies and series and upgrade them if wanted/needed and the feature already exist in them, I think that will be te better place for such function and don't see the need to change my main purpose of the program for this.

**Comment by 5H3RM** [comment]
I do have one suggestion and that is integration into the *arr media manger services. The reason for this as we can use cleanarr to delete the media straight from the media manger thus leaving less chance of media coming back, as opposed to with plex as this could have media re-added if being monitored. Also I'm sure you could use API Calls for deleting media with the *arr service, but don't quote me on it.

Overall this project sounds like something I can get behind! Right now the manual, inaccurate, and pain staking process of manually removing media that is watched very little or not at all. If I'm understanding everything correctly, we will be able to sort media based on criteria, select what we want, and poof, it's gone, which is exactly what I'm looking for! If you need tester I'm here but i'm no coder. 

I think your hardest part will be getting the play count extracted from Tautulli. I know ExporterTools was working on somethings for that but may have gone in another direction.

Good Luck and God Speed!
